routes/orders.js

```
@@ -1,202 +1,177 @@
-    res.json(updated);
+
+    res.json(orders);
   } catch (err) {
-    console.error('Update order error:', err);
     res.status(500).json({ error: err.message });
   }
 });
 
-// 9) Lấy tất cả đơn (GET /orders)
-router.get('/', async (req, res) => {
+router.post('/checkout', async (req, res) => {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)